artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Run.java

```diff
@@ -133,6 +133,8 @@ public void deActivate() {
             broker.components.add(broker.web);
          }
 
+         server.getServer().getConfiguration().parsePrefixedProperties(broker.web, "web", System.getProperties(), ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix());
```

Comment:
think maybe system-web, or even "system-" + ActiveMQDefaultConfiguration.getDefaultWebPropertyPrefix()

this will only show up in the config.getStatus, to identify checksum and to capture any errors specific to that set of properties.

Reply:
that would make it "system-webconfig." am wondering if the dot should be removed, but if the prefix is configured by a user, it is good that the actual configured value appears in the status. So i would keep the dot.

Reply:
I was sort of combining them for the name that identifies where the properties come from. system-<prefix> may get us there. It only shows up in the status.

Reply:
@gtully thanks for your suggestion, I applied it.